release-notes/opensearch-geospatial.release-notes-2.4.0.0.md

```diff
@@ -0,0 +1,25 @@
+## Version 2.4.0.0 Release Notes
+
+Compatible with OpenSearch 2.4.0
+
+### Features
```

Comment:
We need to add XYShape in the Features list

Reply:
Could you point me out the PR. I couldn't find it.

Reply:
I think there are many PRs as each PR merged to main branch
Use this link to filter the PRs https://github.com/opensearch-project/geospatial/pulls?q=is%3Apr+is%3Aclosed+author%3AVijayanB+shape

Reply:
@VijayanB Hi Vijay. Could you check the PR list and make sure that they are back ported to 2.x branch and attach v2.4.0 label accordingly?

Reply:
Updated.